website/docs/d/satellite_location.html.markdown

```diff
@@ -0,0 +1,39 @@
+---
+
+subcategory: "Satellite"
+layout: "ibm"
+page_title: "IBM : satellite_location"
+description: |-
+  Get information about an IBM Cloud satellite location.
+---
+
+# ibm\_satellite_location
+
+Import the details of an existing satellite location as a read-only data source. You can then reference the fields of the data source in other resources within the same configuration by using interpolation syntax.
+
+
+## Example Usage
+
+```hcl
+data "ibm_satellite_location" "location" {
+  location       = "satellite-ibm"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `location` - (Required, string) The name of the location to be created or pass existing location name.
+* `managed_from` - (Required, string) The IBM Cloud metro from which the Satellite location is managed. To list available multizone regions, run 'ibmcloud ks locations'. such as 'wdc04', 'wdc06' or 'lon04'.
```

Comment:
Move all except location to attribute refernce they are computed

Reply:
Moved all attribute to reference except location